deployment/terraform/info.go

```diff
@@ -47,10 +42,13 @@ func displayInfo(output *Output) {
 		}
 	}
 
-	fmt.Println("Load Agent(s):")
-	for _, agent := range output.Agents {
-		fmt.Println("- " + agent.Tags.Name + ": " + agent.PublicIP)
+	if output.HasAgents() {
+		fmt.Println("Load Agent(s):")
+		for _, agent := range output.Agents {
+			fmt.Println("- " + agent.Tags.Name + ": " + agent.PublicIP)
+		}
 	}
+
 	fmt.Println("Coordinator: " + output.Agents[0].PublicIP)
```

Comment:
We likely need to wrap this one or :boom:

Reply:
Hehe nice one